src/webgpu/api/validation/createBindGroup.spec.ts

```diff
@@ -748,14 +748,17 @@ g.test('storage_texture,mip_level_count')
 g.test('buffer,usage')
   .desc(
     `
-    Test that the buffer usage contains 'uniform' if the BindGroup entry defines
-    buffer and it's type is 'uniform'.
+    Test that the buffer usage contains 'UNIFORM' if the BindGroup entry defines buffer and it's
+    type is 'uniform', and the buffer usage contains 'STORAGE' if the BindGroup entry's buffer type
+    is 'storage'|read-only-storage'.
   `
   )
   .params(u =>
     u //
+      .combine('type', ['uniform', 'storage', 'read-only-storage'] as const)
```

Comment:
Use `kBufferBindingTypes` (from `capability_info.ts`) instead to make sure this stays up to date.

Reply:
Ah, I missed checking it. Use it!